src/app/shared/entry-actions/workflow-actions.component.ts

```diff
@@ -65,4 +68,8 @@ export class WorkflowActionsComponent extends EntryActionsComponent implements O
     const prefix = this.sessionQuery.getValue().entryType === EntryType.Service ? ga4ghServiceIdPrefix : ga4ghWorkflowIdPrefix;
     this.refreshService.refreshWorkflow(prefix + this.workflow.full_workflow_path, versionName);
   }
+
+  delete() {
+    this.dialog.open(DeleteEntryDialogComponent, { width: '600px', data: this.workflow });
```

Comment:
Should use one of these sizes:
https://github.com/dockstore/dockstore-ui2/blob/d77d52aff6a67d9f739b91959701abae09c697e7/src/app/shared/constants.ts#L28-L30